packages/image-transform/lib/transform.js

```diff
@@ -107,8 +107,8 @@ const unsafeResizeFromBuffer = async (originalBuffer, options = {}) => {
         // CASE: Automatically remove metadata and rotate based on the orientation.
         .rotate();
 
-    if (options.format) {
-        s = s.toFormat(options.format);
```

Comment:
I think we want to keep this because `options.format` is only used when a forced format conversion is requested.

Ideally we'd be checking the format from the metadata available through sharp to see if we can add the `mozjpeg` option any time we're working with jpeg files rather than only when doing a forced jpeg conversion (which isn't often).